### ISSUE TITLE
It is possible to create multiple micro schemas with the same name

### ISSUE THREAD

**Issue by elbird** [gentics/mesh#310]
## Gentics Mesh Version, operating system, or hardware.
- v0.16.0

## Problem
It is possible to create 2 microschemas with the same name

## Reproducer

HTTP-POST (2x) of payload below to `/api/v1/microschemas` does not cause an exception.
`````
{
	"name": "some_micorschema",
	"fields": [
		{
			"name": "value",
			"required": false,
			"type": "string"
		}
	]
}
`````
## Expected behaviour and actual behaviour
If there is already a microschema with the same name, Mesh should throw an error and stop processing the request. This behavior is already implemented for regular schemas.

**Comment by Jotschi** [comment]
@elbird Please provide a unit testcase for this. See MicroschemaEndpointTest - Thanks.

**Comment by elbird** [comment]
Added test case

**Comment by philippguertler** [comment]
This has been fixed in 0.18.0 https://getmesh.io/docs/changelog/#v0.18.0
